analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SillyBitwiseOperation.cs

```diff
@@ -69,6 +69,18 @@ public static void WithUnaryPrefix()
             result = bitMask | + + +zero;  // FN - Unary Operator is not supported
         }
 
+        public void AfterALoop()
+        {
+            var length = 0x80;
+            var bytesRequired = 0;
+            while (length > 0)
+            {
+                length >>= 8;
+                bytesRequired++;
+            }
```

Comment:
Can be minimized, to isolate the root cause
```suggestion
            var bytesRequired = 0;
            bytesRequired++;
```